core/tauri-utils/src/config.rs

```diff
@@ -262,6 +262,9 @@ pub struct DebConfig {
   /// The files to include on the package.
   #[serde(default)]
   pub files: HashMap<PathBuf, PathBuf>,
+  /// By default, the bundler uses an internal template.
+  /// This option allows you to define your own desktop file.
+  pub desktop_template: Option<PathBuf>,
```

Comment:
hmm, maybe the comment should include that it's looking for a path to a file, not the file contents?
The PathBuf context get's lost in the json schema so it may not be clear enough, what do you think?

Reply:
Just like this, maybe? It should be self-evident that there's a default
```suggestion
  /// Path to a custom desktop file template.
  pub desktop_template: Option<PathBuf>,
```

Reply:
sounds good to me 👍